sources/packages/forms/src/form-definitions/studentdependantsappeal.json

```diff
@@ -583,7 +583,7 @@
                 }
               ],
               "tableView": false,
-              "label": "<strong>Dependants</strong>",
+              "label": "Dependants",
```

Comment:
Are we removing the strong because the labels are by default bold / strong now after the formio upgrade?

Reply:
They were already bold before the upgrade, there was no need to have them added.
After the upgrade, the validation messages started to display them inside the message.
Please see the previous PR description for further information: https://github.com/bcgov/SIMS/pull/3731